### PR TITLE
[IMP] web_hierarchy: add filters in the org chart view

### PR DESCRIPTION
**Pull request by hugocarlier**
task-3582032

### CODE REVIEW

File: addons/web/static/src/search/search_bar/search_bar.xml [hunk 1]
```diff
@@ -93,7 +93,7 @@
 
     <t t-name="web.SearchBar">
         <div class="o_cp_searchview d-flex input-group" role="search" t-ref="root">
-            <div class="o_searchview form-control d-print-contents d-flex align-items-center py-1" role="search" aria-autocomplete="list">
+            <div class="o_searchview form-control d-print-contents d-flex align-items-center py-1 border-end-0" role="search" aria-autocomplete="list">
```
Comment (Bertrand2) [inline addons/web/static/src/search/search_bar/search_bar.xml:96]: :innocent: 
```suggestion
            <div class="o_searchview form-control d-print-contents d-flex align-items-center py-1 border-end-0"
                role="search" aria-autocomplete="list">
```

**Review by Bertrand2 — APPROVED** [review]
Just a minor nitpicking, otherwise LGTM :+1:

**Comment by sofiagvaladze** [comment]
rubodoo r+

**Comment by huca-odoo** [comment]
> rubodoo r+

Thank you @sofiagvaladze but I think there is a typo here :sweat_smile:

**Comment by sofiagvaladze** [comment]
robodoo r+

**Comment by sofiagvaladze** [comment]
robodoo rebase-ff

**Pull request closed without merging**